docs/spec/relayer/Relayer.tla

```diff
@@ -155,14 +77,17 @@ LightClientUpdates(srcChainID, dstChainID, relayer) ==
 \* Compute connection datagrams designated for dstChainID. 
 \* These are used to update the connection end on dstChainID in the environment.
 ConnectionDatagrams(srcChainID, dstChainID) ==
-    LET srcConnectionEnd == GetConnectionEnd(srcChainID) IN
-    LET dstConnectionEnd == GetConnectionEnd(dstChainID) IN
+    LET srcChain == GetChainByID(srcChainID) IN
+    LET dstChain == GetChainByID(dstChainID) IN
+    
+    LET srcConnectionEnd == GetConnectionEnd(srcChain) IN
+    LET dstConnectionEnd == GetConnectionEnd(dstChain) IN
 
     LET srcConnectionID == GetConnectionID(srcChainID) IN
     LET dstConnectionID == GetConnectionID(dstChainID) IN
 
-    LET srcHeight == GetLatestHeight(srcChainID) IN
-    LET srcConsensusHeight == GetMaxCounterpartyClientHeight(srcChainID) IN
+    LET srcHeight == GetLatestHeight(srcChain) IN
+    LET srcConsensusHeight == GetMaxCounterpartyClientHeight(srcChain) IN
     
     IF dstConnectionEnd.state = "UNINIT" /\ srcConnectionEnd.state = "UNINIT" THEN 
          {[type |-> "ConnOpenInit", 
```

Comment:
I am not sure if we want to create ConnOpenInit the same way as other datagrams as this is the initial trigger that should come from outside, for example from the config file. Don't have better proposal, but leaving comment as a reminder to discuss.